canonical-websites/www.ubuntu.com
# Fixed 404 links in the /legal section

## Done

- Updated links
- **WAITING ON LEGAL FOR DIRECTION ON A FEW MORE, SO DON'T MERGE**

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
    -  [/legal/data-privacy/2013-03-25/](http://0.0.0.0:8001/legal/data-privacy/2013-03-25/)
    -  [/legal/data-privacy/2016-02-08/](http://0.0.0.0:8001/legal/data-privacy/2016-02-08/)
    -  [/legal/data-privacy/third-parties](http://0.0.0.0:8001/legal/data-privacy/third-parties)
    -  [/legal/data-privacy/font-licence/faq](http://0.0.0.0:8001/legal/font-licence/faq)

- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Test the links, I will remake the copy docs afterwards 


## Issue / Card

Fixes #4657
